packages/dashboards/app/styles/navi-dashboards/components/navi-widget.less

```diff
@@ -25,28 +25,11 @@
     font-size: @font-size-xlarge;
   }
 
-  &__action {
-    font-size: @font-size-base-large;
-    color: @body-font-gray;
-  }
-
-  &__delete-btn {
-    .btn {
-      font-size: @font-size-base-large;
-    }
-  }
-
   &__filter-errors-icon {
     color: @navi-yellow;
     font-size: @font-size-base-large;
   }
 
-  &__actions {
```

Comment:
Something in this file affected the widget edit/delete button

Reply:
addressed